python_modules/dagster/dagster/_core/definitions/antlr_asset_selection/AssetSelection.g4

```diff
@@ -82,6 +84,7 @@ ROOTS: 'roots';
 // String tokens
 QUOTED_STRING: '"' (~["\\\r\n])* '"';
 UNQUOTED_STRING: [a-zA-Z_][a-zA-Z0-9_]*;
+UNQUOTED_REGEX_STRING: [a-zA-Z_*][a-zA-Z0-9_*]*;
```

Comment:
doesn't seem like this is full regex, just wildcard, if we don't support e.g. `+`? or do we only allow a subset of regex for unquoted strings?

Reply:
Ah yeah we only support wildcard. I'll rename this to UNQUOTED_WILDCARD_STRING.